matlab/vts_wrapper/vts_solver_demo.m

```diff
@@ -343,14 +343,14 @@
 
 % % or 
 scatterer.Type = 'Intralipid';
-scatterer.vol_frac =  0.5;
+scatterer.vol_frac =  0.01;
 
 % % or 
 % scatterer.Type = 'Mie';
 % scatterer.radius =  0.5;
 % scatterer.n =       1.4;
 % scatterer.nMedium = 1.0;
-% scatterer.vol_frac = 0.5;
+% scatterer.vol_frac = 0.001;
```

Comment:
Just curious, why did you modify some of the values in the MATLAB?

Reply:
Some of the Matlab values were bad - didn't mimic tissue at all. They're consistent with the C# side.